src/mono/wasm/debugger/BrowserDebugProxy/DevToolsProxy.cs

```diff
@@ -388,4 +398,48 @@ protected void Log(string priority, string msg)
             }
         }
     }
+    public class FirefoxProxyServer
```

Comment:
- put new classes in separate files.
- use file-scoped namespaces in them, `namespaces foo;`
- Use `#nullable enable`

Reply:
done.